lemminx-maven/src/main/java/org/eclipse/lemminx/maven/SubModuleValidator.java

```diff
@@ -34,21 +32,15 @@ public void setPomFile(File pomFile) throws FileNotFoundException, IOException,
 
 	public Diagnostic validateSubModuleExistence(DiagnosticRequest diagnosticRequest) {
```

Comment:
This could also be an `Optional<Diagnostic>`, or even an `Optional<List<Diagnostic>>` so this could be referenced in MavenDiagnosticParticipant.tagDiagnostics.